lib/modules/hosts/bime.js

```diff
@@ -0,0 +1,14 @@
+/* eslint no-unused-vars: [2, { "args": "none" }] */
```

Comment:
This is only necessary in the example host (not here) because it doesn't do anything. I'll change that to make it more intuitive.